docs/src/pages/components/Tables/WebTables.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Tables
+platform: web
+githubPath: bpk-component-table
+---
+
+import {
+  BpkTable,
+  BpkTableHead,
+  BpkTableBody,
+  BpkTableRow,
+  BpkTableHeadCell,
+  BpkTableCell,
+} from 'bpk-component-table';
+
+import Readme from 'components/Readme';
+import PresentationBlock from 'components/PresentationBlock';
+import tablesReadme from 'bpk-component-table/README.md';
+import datatableReadme from 'bpk-component-datatable/README.md';
+
+import { AutowidthExample } from 'bpk-component-datatable/examples';
+
+## Table of contents
+
+## Standard table
+
+A simple table component for displaying data in a tabular form.
+
+<PresentationBlock>
+  <BpkTable>
+    <BpkTableHead>
+      <BpkTableRow>
+        <BpkTableHeadCell>Heading 1</BpkTableHeadCell>
+        <BpkTableHeadCell>Heading 2</BpkTableHeadCell>
+        <BpkTableHeadCell>Heading 3</BpkTableHeadCell>
+        <BpkTableHeadCell>Heading 4</BpkTableHeadCell>
+      </BpkTableRow>
+    </BpkTableHead>
+    <BpkTableBody>
+      <BpkTableRow>
+        <BpkTableCell>Row 1, Data 1</BpkTableCell>
+        <BpkTableCell>Row 1, Data 2</BpkTableCell>
+        <BpkTableCell>Row 1, Data 3</BpkTableCell>
+        <BpkTableCell>Row 1, Data 4</BpkTableCell>
+      </BpkTableRow>
+      <BpkTableRow>
+        <BpkTableCell>Row 2, Data 1</BpkTableCell>
+        <BpkTableCell>Row 2, Data 2</BpkTableCell>
+        <BpkTableCell>Row 2, Data 3</BpkTableCell>
+        <BpkTableCell>Row 2, Data 4</BpkTableCell>
+      </BpkTableRow>
+    </BpkTableBody>
+  </BpkTable>,
+</PresentationBlock>
```

Comment:
We're trying to stick to making the code samples in mdx very simple – I'm guessing this long one is because on the web repo side you've only migrated datatable and not table so you don't have an example to import? Which I think is fine, we can replace it later.

Reply:
Yes this component has not been updated to the new format yet - I was thinking of leaving this but the datatable example needed to change on the docs side and given these are both on the same page it was difficult to not do it when the `BpkTable` has not been updated yet